RealtimeStreamToImpala/README.rst

```diff
@@ -0,0 +1,54 @@
+Realtime Stream To Impala Adapter Configuration
```

Comment:
Will be using https://github.com/cdap-guides/cdap-etl-adapter-guide/pull/1 as the format. Made a comment about this in https://github.com/cdap-guides/cdap-etl-adapter-guide/pull/1